### PR TITLE
closes #71

### LINKED ISSUE #71: Modify Product API

### PR DESCRIPTION
**Pull request by chloe630**
closes #71

### CODE REVIEW

**Review by JeffKandel — CHANGES_REQUESTED** [review]
The POST tests are going to fail, they try to create a user and not a review.  However, I think it's better to have the product API in our master branch and deal with testing later.  See comment below.

File: server/review.test.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+const request = require('supertest')
+  , {expect} = require('chai')
+  , db = require('APP/db')
+  , app = require('./start')
+
+/* global describe it before afterEach */
+
```
Comment (JeffKandel) [inline server/review.test.js:7]: Please add the following comment!

// To do: These test need to be re-written at a later point.

**Review by JeffKandel — CHANGES_REQUESTED** [review]
Added a comment.

File: server/review.test.js (hunk 1 above)
Comment (chloe630) [reply server/review.test.js:7]: to prevent any conflict, I will add this on review-test branch!

**Review by lada-tech — APPROVED** [review]
just merge it, we'll deal with it later

**Review by gabrielwr — APPROVED** [review]

File: server/review.test.js (hunk 1 above)
Comment (gabrielwr) [reply server/review.test.js:7]: Merged. This comment will be added to Chloe's current branch dealing with these tests.

**Pull request merged**